src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -530,7 +535,7 @@ async Task<CleanEntityStorageResult> IDurableEntityClient.CleanEntityStorageAsyn
                     }
 
                     if (removeEmptyEntities && !status.EntityExists && status.LockedBy == null && status.QueueSize == 0
-                        && now - state.LastUpdatedTime > TimeSpan.FromMinutes(this.config.Options.EntityMessageReorderWindowInMinutes))
+                        && now - state.LastUpdatedTime > this.config.MessageReorderWindow)
```

Comment:
nitpick: since this is such a long conditional check, I would recommend splitting it into smaller, well-named, boolean variables. It would improve readability and help us detect bugs in the future

Reply:
Yes, I agree. I am splitting this apart now.